survey/templates/survey/questiondetail.html

```diff
@@ -165,7 +165,9 @@ <h3 align="center"> {{ page.page_title }} </h3>
     
                             {% elif answer.atype == 'Tf' %}
                                 <textarea rows="5" cols="60" name="{{ q.id }}_{{ answer.id }}" id="{{ answer.id }}" value="{{ resp }}">{{ answer.atext }}</textarea><br />
-                            
+                            {% elif answer.atype == 'Tx' %}
+                                <label class="answer">
```

Comment:
Unclosed label tag.

Might want to just use an <input></input> tag rather than the XML style <input />, and wrap the answer text in that.